fabric-chaincode-shim/src/test/java/org/hyperledger/fabric/ledger/LedgerTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020 IBM All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.hyperledger.fabric.ledger;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+
+import org.hyperledger.fabric.contract.Context;
+import org.junit.jupiter.api.Test;
+
+public class LedgerTest {
+
```

Comment:
Would be good to add tests to document/check that new instances are created every time